crates/meilisearch-types/src/tasks.rs

```diff
@@ -607,6 +622,9 @@ pub enum Details {
     IndexSwap {
         swaps: Vec<IndexSwap>,
     },
+    UpgradeDatabase {
+        from: (usize, usize, usize),
```

Comment:
Would be great to also have `to: (usize, usize, usize)` here, so that we can track from which to which versions a DB has been upgraded

Reply:
But we're always upgrading to the latest version, right?
If we add this information, we should update the details of the task, every time we restart the task I guess? :thinking:

Reply:
If I look at an upgrade task I did 2 weeks ago I might be happy to know if it was targeting v1.13.2 or v1.13.9

Reply:
So from what I understand, you would prefer the `to` to be created once when it's enqueued and then never change again?